specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -617,6 +756,92 @@
           "type": "string"
         }
       }
+    },
+    "PutSubscriptionRequest": {
+      "description": "The parameters required to create a new subscription.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PutSubscriptionRequestProperties",
+          "description": "Put subscription request properties."
+        }
+      }
+    },
+    "PutSubscriptionRequestProperties": {
+      "description": "Put subscription properties.",
+      "required": [
+        "displayName",
+        "skuId",
+        "billingScope"
+      ],
+      "properties": {
+        "displayName": {
+          "description": "The friendly name of the subscription.",
+          "type": "string"
+        },
+        "skuId": {
+          "description": "The SKU ID of the Azure plan.",
+          "type": "string"
+        },
+        "billingScope": {
+          "description": "Determines customerLed / fieldLed, partnerLed.",
+          "type": "string"
+        },
+        "initialOwnerPrincipalId": {
+          "description": "Initial owner object Id.",
+          "type": "string"
+        },
+        "initialOwnerTenantId": {
+          "description": "Initial owner tenant Id.",
+          "type": "string"
+        }
+      }
+    },
+    "PutSubscriptionResponse": {
+      "description": "Subscription information (same as Microsoft.Resources/subscription).",
+      "properties": {
+        "id": {
+          "description": "Fully qualified ID for the subscription.",
```

Comment:
Just to differentiate, what about "Fully qualified ID for the subscription resource."